### PR TITLE
Filtering fixes

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/controllers/concerns/palace_attendees_mixin.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  def update
+    authorize form_answer, :update?
+
+    palace_attendee = palace_invite.palace_attendees.find(params[:id])
+    palace_attendee.update(create_params)
+    render_attendee_form(palace_attendee)
+  end
+
+  def destroy
```
Comment (matthewford) [inline app/controllers/concerns/palace_attendees_mixin.rb]: Assignment Branch Condition size for destroy is too high. [15.03/15]

File: app/models/form_answer.rb [hunk 2]
```diff
@@ -49,6 +49,7 @@ class FormAnswer < ActiveRecord::Base
     has_one :press_summary, dependent: :destroy
     has_one :draft_note, as: :notable, dependent: :destroy
     has_one :company_detail, dependent: :destroy
+    has_one :palace_invite, dependent: :destroy
```
Comment (matthewford) [inline app/models/form_answer.rb]: Inconsistent indentation detected.

File: app/search/form_answer_search.rb [hunk 3]
```diff
@@ -51,6 +51,14 @@ def filter_by_sub_status(scoped_results, value)
         out = out.joins(
           "LEFT OUTER JOIN press_summaries on press_summaries.form_answer_id = form_answers.id"
         ).where("press_summaries.id IS NULL OR press_summaries.approved = false")
+      when "missing_rsvp_details"
+        out = out.joins(
+          "LEFT OUTER JOIN palace_invites on palace_invites.form_answer_id = form_answers.id"
+        ).joins(
+          "LEFT OUTER JOIN palace_attendees ON palace_attendees.palace_invite_id = palace_invites.id"
```
Comment (matthewford) [inline app/search/form_answer_search.rb]: Line is too long. [101/100]

File: spec/search/form_answer_search_spec.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    expect(res).to include(invite2)
+  end
+
+  it "filters by corp_responsibility missing" do
+    admin = create(:admin)
+    create(:form_answer)
+    form2 = create(:form_answer, document: {"corp_responsibility_form" => "declare_now"})
+    scope = FormAnswer.all
+    expect(FormAnswer.count).to eq(2)
+    res = described_class.new(scope, admin).filter_by_sub_status(scope, ["missing_corp_responsibility"])
```
Comment (matthewford) [inline spec/search/form_answer_search_spec.rb]: Line is too long. [104/100]

File: spec/search/form_answer_search_spec.rb [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    expect(res).to_not include(invite2)
+    res = described_class.new(scope, admin).filter_by_sub_status(scope, [])
+    expect(res.size).to eq(4)
+    expect(res).to include(invite2)
+  end
+
+  it "filters by corp_responsibility missing" do
+    admin = create(:admin)
+    create(:form_answer)
+    form2 = create(:form_answer, document: {"corp_responsibility_form" => "declare_now"})
```
Comment (matthewford) [inline spec/search/form_answer_search_spec.rb]: Space inside { missing.<br>Space inside } missing.

**Comment by RuslanHamidullin** [comment]
Good :+1:

**Pull request merged**